.github/workflows/lint.yml

```diff
@@ -0,0 +1,15 @@
+name: Lint
+on: [push, pull_request]
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v0.1.7
+        with:
+          # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
+          version: v1.26
```

Comment:
v.1.27

Reply:
@slaskawi it's been a while since I merged this on Louketo, but I recall I had some issues with v1.27. But if that's what you would like, I don't mind.

Honestely I'd suggest to merge it, so you guys can iterate over it.